en/getting_started/sensor_selection.md

```diff
@@ -1,61 +1,60 @@
 # Sensor Selection
 
-PX4 autopilot is capable of flying without any additional external
-sensors, because Pixhawk (the hardware FC) already has inbuilt sensors.
-There are 3 Gyros + 3 Accelerometers + 3 Magnetometers + Barometric
-pressure sensor for proper altitude reading. Despite that, we strongly
-recommend using **GPS** in all frames in order to unveil the full
-potential of the aircraft or vehicle and enable its automatic modes. For
-most of the fixed wing aircraft, **Airspeed sensor** is mandatory as
-well.
+PX4-based systems use sensors to determine vehicle state (state is required for stabilization and to enable autonomous control). The sensors provide information including: position/altitude, heading, speed, airspeed, orientation (attitude) and rates of rotation in different directions.
+
+The system *minimally requires* a gyroscope, accelerometer, magnetometer (compass) and barometer. A GPS or other positioning system is needed to enable all automatic [modes](../getting_started/flight_modes.md#categories), and some assisted modes. Fixed wing and VTOL-vehicles should additionally include an airspeed sensor (very highly recommended).

```

Comment:
Is an airspeed sensor *manadatory* for FW and VTOL? My understanding is that it is very highly recommended for the VTOL transition but not mandatory. By inference, it should then also not be mandatory for FW.

What are your thoughts on level of need? - "recommended", "highly recommended", "near mandatory", "mandatory" or "required"